ysoserial/packages.config

```diff
@@ -1,15 +1,40 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="AspNetTicketBridge" version="1.6.0" targetFramework="net472" />
```

Comment:
Remove all these imported libraries. Just add a reference for System.IdentityModel.Services